### PR TITLE
[ticket/15237] Fix unguarded includes to functions_user

### PR DESCRIPTION
**Pull request by rmcgirr83**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

https://tracker.phpbb.com/browse/PHPBB3-15237

PHPBB3-15237

### CODE REVIEW

File: phpBB/includes/mcp/mcp_ban.php [hunk 1]
```diff
@@ -28,7 +28,10 @@ function main($id, $mode)
...
 		global $phpbb_root_path, $phpEx;
 
-		include($phpbb_root_path . 'includes/functions_user.' . $phpEx);
+		if (!function_exists('user_ban'))
+		{
+			include($phpbb_root_path . 'includes/functions_user.' . $phpEx);
+		}
 
 		// Include the admin banning interface...
 		include($phpbb_root_path . 'includes/acp/acp_ban.' . $phpEx);
```
Comment (javiexin) [inline phpBB/includes/mcp/mcp_ban.php]: Wouldn't this have to be guarded as well?

Comment (marc1706) [inline phpBB/includes/mcp/mcp_ban.php]: Usually no, as admin modules shouldn't be included manually (unless in this exact case where we have some annoying legacy code).

Comment (rmcgirr83) [inline phpBB/includes/mcp/mcp_ban.php]: @marc1706 anything else I need to do to this or is it mergeful?

Comment (marc1706) [inline phpBB/includes/mcp/mcp_ban.php]: Question answered? :)

Comment (rmcgirr83) [inline phpBB/includes/mcp/mcp_ban.php]: :grin:  Gracias!

**Pull request merged**